doc/man/cgxget.1

```diff
@@ -43,6 +43,11 @@ If the system is running in cgroup v1 mode, libcgroup will convert the data as n
 .B -a, --all
 print the variables for all controllers which consist in the  given cgroup
 
+.TP
+.B -b
+Ignores the default systemd delegated hierarchy path and constructs the path of the control groups
```

Comment:
Same capitalization question.  Ironically, it looks like `cgdelete.1` capitalized every entry.  Weird.